config/registry-config.yaml

```diff
@@ -0,0 +1,42 @@
+# https://docs.docker.com/registry/configuration/
+
+version: 0.1
+log:
+  level: debug
+  fields:
+    service: registry
+storage:
+  cache:
+    blobdescriptor: inmemory # or redis?
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 443:5000 # Why the 443 -> https://docs.docker.com/registry/configuration/#letsencrypt
```

Comment:
Are you asking here or documenting?

Reply:
Both I guess. I know you wanted to use `let's encrypt` and that's what the documentation says to do.